src/ui/views/chrome.jade

```diff
@@ -10,6 +10,37 @@ html(lang='en')
     meta(http-equiv='X-UA-Compatible', content='IE=edge,chrome=1')
     meta(name='viewport', content='width=device-width')
     title Kibana
+    style.
+      @font-face {
+        font-family: 'Open Sans';
+        font-style: normal;
+        font-weight: 300;
+        src: local('Open Sans Light'), local('OpenSans-Light'),
+             url('bundles/fonts/open_sans/open_sans_v13_latin_300.woff2') format('woff2'),
```

Comment:
These urls need to be absolute and include the basePath or they won't work at both `/login` and `/app/xyz`.